Tests/Unit/MSFT_xDNSServerAddress.Tests.ps1

```diff
@@ -64,12 +64,39 @@ InModuleScope MSFT_xDNSServerAddress {
                 $Result.IPAddress | Should Be $Splat.IPAddress
             }
         }
+
+        #region Mocks
+        Mock Get-DnsClientServerAddress -MockWith {
+
+            [PSCustomObject]@{
+                ServerAddresses = 'fe80:ab04:30F5:002b::1'
+                InterfaceAlias = 'Ethernet'
+                AddressFamily = 'IPv6'
+            }
+        }
+        #endregion
+
+        Context 'comparing IPv6 Address' {
+            It 'should return true' {
```

Comment:
I added some more relevant tests to the Set/Test-TargetResource functions but I didn't to the get-targetresource because the ServerAddresses parameter is actually ignored by this function so I didn't think that adding the additional tests (with multiple addresses) would do anything. But I might be misunderstanding :)